components/camera/videosourcewrappers.go

```diff
@@ -136,7 +135,6 @@ func NewVideoSourceFromReader(
 		rtpPassthroughSource: rtpPassthroughSource,
 		system:               actualSystem,
 		videoSource:          vs,
-		videoStream:          gostream.NewEmbeddedVideoStream(vs),
```

Comment:
videoStream is unnecessary

Reply:
I think the NextPointCloud in question is this projector provider thing that takes a depth image and projects it to a PCD using intrinsics, a feature we want to deprecate or modularize (that nothing really uses). I'm also unsure how to test it...